go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/installer
 
-go 1.20
+go 1.21
```

Comment:
I don't think this is possible, because 4.15 is compiled with go 1.20:
https://github.com/openshift/installer/blob/release-4.15/.ci-operator.yaml#L4